src/components/Pagination.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button, Center } from "@chakra-ui/react";
+import React from "react";
+
+interface PaginationProps {
+  paginationPages: number;
```

Comment:
`paginationPages` please name it `totalPages`. It will be much easier to understand what it is,